src/asm/section.c

```diff
@@ -1016,3 +1016,27 @@ void sect_PopSection(void)
 	sectionStack = entry->next;
 	free(entry);
 }
+
+bool sect_IsSizeKnown(char const *name)
+{
+	struct Section *sect = sect_FindSectionByName(name);
```

Comment:
Why perform the lookup here instead of letting the caller do it? In the currently single call site, the pointer is already known, so this duplicates work.

Reply:
The `sect_IsSizeKnown` call does do a second `sect_FindSectionByName` lookup, but I think it's a worthwhile tradeoff for a better API. None of the other public `sect_*` functions take a raw `struct Section *` argument. (And section.c is the best place for `sect_IsSizeKnown` since `sectionStack` and `currentLoadSection` are visible there.)

Reply:
If this were implemented in an object-oriented fashion, `->isSizeKnown()` would be an attribute of the section itself, which would translate to a `struct Section *` in C.

Further, no [public `section.h` function](https://github.com/gbdev/rgbds/blob/6d2db2ef642310d7dc39679489adb79ebca5c18f/include/asm/section.h#L45-L77) takes a section name argument, except the section creation ones for an obvious reason.
However, [plenty of `symbol.h` function operate on symbols OO-style](https://github.com/gbdev/rgbds/blob/6d2db2ef642310d7dc39679489adb79ebca5c18f/include/asm/symbol.h#L61-L111), so using a `struct Section const *` here would be consistent with the codebase as a whole.

Reply:
Alright, that makes sense.